### PR TITLE
feat: Add get option contracts

### PR DESCRIPTION
**Pull request by hiohiohio**
Context:
- a new api endpoint, options/contracts are introduced

Changes:
- support options contracts api endpoint

### CODE REVIEW

File: alpaca/trading/models.py [hunk 1]
```diff
@@ -604,3 +606,61 @@ class TradeUpdate(BaseModel):
...
     qty: Optional[float] = None
+
+
+class OptionContract(BaseModel):
+    """
+    Represents an option contract.
+
+    Attributes:
+        id (str) The unique identifier of the option contract.
+    symbol (str): The symbol representing the option contract.
```
Comment (matebudai) [inline alpaca/trading/models.py:617]: indentation

File: alpaca/trading/client.py [hunk 2]
```diff
@@ -661,3 +664,53 @@ def get_corporate_announcement_by_id(
...
+            OptionContracts (Union[OptionContractsResponse, RawData]): The object includes list of option contracts.
+        """
+        if request is None:
+            raise ValueError("request (GetOptionContractsRequest) is required")
+        if request.underlying_symbol == "":
+            raise ValueError("underlying_symbol is required")
+
+        params = request.to_request_fields()
+
+        response = self.get(f"/options/contracts", params)
```
Comment (matebudai) [inline alpaca/trading/client.py:689]: f-string that does not have any interpolated variable

**Review by matebudai — APPROVED** [review]

**Review by AlexandrosKyriakakis — APPROVED** [review]
Looks Good! Just take care the comments from @matebudai

File: alpaca/trading/models.py (hunk 1 above)
Comment (hiohiohio) [reply alpaca/trading/models.py]: Thank you! Fixed!

File: alpaca/trading/client.py (hunk 2 above)
Comment (hiohiohio) [reply alpaca/trading/client.py]: Thank you! Fixed!

**Pull request merged**